jest.config.js

```diff
@@ -10,22 +16,10 @@ module.exports = {
   coverageReporters: ['html', 'text-summary', 'json-summary'],
   coverageThreshold: {
     global: {
-      branches: 44,
-      functions: 46,
-      lines: 52,
-      statements: 52,
-    },
-    './app/scripts/controllers/permissions/**/*.js': {
-      branches: 100,
-      functions: 100,
-      lines: 100,
-      statements: 100,
-    },
-    './app/scripts/lib/createRPCMethodTrackingMiddleware.js': {
-      branches: 95.65,
-      functions: 100,
-      lines: 100,
-      statements: 100,
```

Comment:
Did we want to drop these exceptions? @PeterYinusa? I can do that in #16455 -- currently the strategy I have there is I collect the coverage from all test runners and then combine the coverage reports programatically. Then I check the sub paths like these to ensure coverage remains accurate. I can however simplify our coverage reporting by removing these exceptions if that is preferable.

Reply:
NO we do not want to drop these.
I just dropped these as jest it up only looked at the global coverage and not the coverage in relative/custom paths